packages/database/src/models/Access.ts

```diff
@@ -0,0 +1,76 @@
+import { DataTypes } from 'sequelize';
+
+import { SYNC_DIRECTIONS } from '@tamanu/constants';
+
+import { Model } from './Model';
+import type { InitOptions, Models } from '../types/model';
+
+export class Access extends Model {
```

Comment:
```suggestion
export class AccessLog extends Model {
```

then you can do `tableName: 'accesses'` in the options